src/Microsoft.OData.Client/DataServiceClientResponsePipelineConfiguration.cs

```diff
@@ -309,6 +309,41 @@ internal void ExecuteOnNavigationEndActions(ODataNestedResourceInfo link)
             }
         }
 
+        /// <summary>
+        /// Executes the OData delta resource set start actions.
+        /// </summary>
+        /// <param name="feed">The feed.</param>
+        internal void ExecuteOnFeedStartActions(ODataDeltaResourceSet feed)
+        {
+            Debug.Assert(feed != null, "feed != null");
+            if (this.readingStartFeedActions.Count > 0)
+            {
+                ReadingDeltaResourceSetArgs args = new ReadingDeltaResourceSetArgs(feed);
+                foreach (Action<ReadingDeltaResourceSetArgs> feedAction in this.readingStartFeedActions)
+                {
+                    feedAction(args);
+                }
+            }
+        }
+
+        /// <summary>
+        /// Executes the OData delta resource set end actions.
+        /// </summary>
+        /// <param name="deltaResourceSet">The OData delta resource set.</param>
+        internal void ExecuteOnFeedEndActions(ODataDeltaResourceSet deltaResourceSet)

```

Comment:
Choose either `deltaFeed` or `deltaResourceSet` as the parameter name for both `ExecuteOnFeedStartActions` and `ExecuteOnFeedEndActions`